packages/cli/src/daemon/log-requests.ts

```diff
@@ -2,10 +2,38 @@ import { LoggerProvider } from '@ceramicnetwork/common'
 import { Request, Response } from 'express'
 import morgan from 'morgan'
 
+const EXPECTED_RESPONSE_TIME_MS = 2000
+const MAX_DAILY_SLOW_REQUESTS_TO_LOG = 10
```

Comment:
What about only logging one slow query per minute instead?  I think that will still be infrequent enough to not flood the logs, while making the slow queries a lot more visible if they are happening often.